src/tools/crawl.ts

```diff
@@ -1,20 +1,27 @@
+// Comment what this file does, and overall add more comment to every file and blocks of code.
+
 import * as fg from 'fast-glob';
 import * as fs from 'fs';
 import * as path from 'path';
 
 import { IGroupDesc } from '../framework/loader.js';
 
+// Could probably be a constant exposed by loader.js
```

Comment:
Doesn't really make sense because `loader.js` sees `.spec.js` files while this runs under node and sees `.spec.ts` files.

Reply:
Ack